bang/magicollections.py

```diff
@@ -696,6 +696,27 @@ def to_fields(self, item, extra_fields=None, exclude_fields=None, order=None, *a
                         'link_text': cameo.name,
                     } for cameo in item.cached_cameos]
                 }))
+            # Add live2d viewer
+            if hasattr(item, "associated_costume"):
```

Comment:
It will always return True:

```python
>>> hasattr(models.Song.objects.filter(event__isnull=True).select_related('event')[0], 'event')
True
>>> hasattr(models.Song.objects.filter(event__isnull=True)[0], 'event')
True
```

-->

```python
if item.associated_costume:
```

Reply:
If I do that it still raises AttributeError for cards without costumes, so I've left this one the way it is for now.